src/test/java/pl/szczesniak/dominik/whattowatch/users/domain/model/UserServiceInMemoryTest.java

```diff
@@ -0,0 +1,53 @@
+package pl.szczesniak.dominik.whattowatch.users.domain.model;
+
+import org.junit.jupiter.api.Test;
+import pl.szczesniak.dominik.whattowatch.users.domain.model.exceptions.UserDoesNotExistException;
+import pl.szczesniak.dominik.whattowatch.users.infrastructure.persistence.InMemoryUserRepository;
+import static pl.szczesniak.dominik.whattowatch.users.domain.model.TestUserServiceConfiguration.userService;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+
+
+
+class UserServiceInMemoryTest {
+
+    private final UserService tut = userService(new InMemoryUserRepository());
+
+    @Test
+    void should_return_true_when_user_exists() {
+        // when
+        tut.createUser("Dominik");
+
+        // then
+        assertThat(tut.exists(tut.getUserId("Dominik"))).isTrue();
+    }
+
+    @Test
+    void should_throw_exception_user_does_not_exist() {
+        // when
+        final Throwable thrown = catchThrowable(() -> tut.exists(tut.getUserId("Dominik")));
+
+        // then
```

Comment:
A ten test jest już ewidentnie dziwny. Dlaczego rzucasz wyjątek pytając o to czy użytkownik istnieje. Powinniśmy dostać false jeśli nie istnieje, a nie wyjątek